gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -314,7 +314,7 @@ private Gradebook getGradebook(final String siteId) {
 
 		List<CategoryDefinition> rval = new ArrayList<>();
 
-		if (gradebook != null) {
+		if (gradebook != null && categoriesAreEnabled()) {
```

Comment:
As categories are retained in the database when categories are disabled for a gradebook, I've changed this to only return categories when categories are enabled.